crates/wallet/src/wallet/params.rs

```diff
@@ -107,7 +107,12 @@ impl CreateParams {
         self
     }
 
-    /// Use custom lookahead value.
+    /// Use a custom `lookahead` value.
+    ///
+    /// The `lookahead` defines a number of script pubkeys to derive over and above the last
+    /// revealed index. Without a lookahead the indexer will miss outputs you own when processing
+    /// transactions whose output script pubkeys lie beyond the last revealed index. In most cases
+    /// the default value [`DEFAULT_LOOKAHEAD`] is sufficient.
```

Comment:
question: should the lookahead explanation/definition at [`KeychainTxOutIndex`](https://github.com/bitcoindevkit/bdk/blob/8760653339d3a4c66dfa9a54a7b9d943a065f924/crates/chain/src/indexer/keychain_txout.rs#L48-L58) be updated too ?

Reply:
I think that's an accurate description as well. I took these docs almost verbatim from `KeychainTxOutIndex::new`. Indeed there's room to improve the overall consistency of the docs